pegasus/sites.v3/code.org/public/css/tutorial-landing.css

```diff
@@ -48,6 +48,12 @@
   padding: 10px;
 }
 
+@media screen and (min-width: 513px) and (max-width: 970px) {
```

Comment:
this css is only referenced from pegasus/sites.v3/code.org/public/playlab.haml (pegasus does not include all css on all pages)